storage/plugins/archives/ArchivesPlugin.php

```
@@ -3980,6 +4076,46 @@ private function validateArchivalUnit(array $values, ?int $excludeId = null): ar
+        } elseif ($ark !== '') {
+            $sql = 'SELECT id FROM archival_units WHERE ark_identifier = ? AND deleted_at IS NULL';
+            if ($excludeId !== null) {
+                $sql .= ' AND id != ?';
+            }
+            $arkCheck = $this->db->prepare($sql);
+            if ($arkCheck !== false) {
+                if ($excludeId !== null) {
+                    $arkCheck->bind_param('si', $ark, $excludeId);
+                } else {
+                    $arkCheck->bind_param('s', $ark);
+                }
```

PHPMD (2.15.0)

[error] The method validateArchivalUnit uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)